src/components/Map/Map.jsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState, useContext, useEffect } from "react";
+import OpenLayersMap from "./OpenLayersMap";
+import { TileLayer, VectorLayer, MapboxLayer, MaskLayer } from "./Layers";
+import { Style, Icon, Fill, Stroke, Circle, Image } from "ol/style";
+import Feature from "ol/Feature";
+import Point from "ol/geom/Point";
+import { osm, vector, mask } from "./Source";
+import { fromLonLat, get } from "ol/proj";
+import GeoJSON from "ol/format/GeoJSON";
+import SDN_ADM0 from "./Data/sdn_adm0.json";
+import SDN_ADM1 from "./Data/sdn_adm1.json";
```

Comment:
Why are we importing the jsons on Map component. This should be passed from the stories.